tests/roots/test-footnotes-html/conf.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please rename this to more suitable directory name.

Reply:
i'd appreciate a proposal.